app/index.html

```diff
@@ -23,19 +23,9 @@
     <script src="bower_components/angular/angular.js"></script>
     <script src="bower_components/angular-messages/angular-messages.js"></script>
     <script src="bower_components/moment/moment.js"></script>
-    <script src="bower_components/angular-form-messages/src/afModule.js"></script>
-    <script src="bower_components/angular-form-messages/src/afMessages.js"></script>
-    <script src="bower_components/angular-form-messages/src/bootstrap/afModule.js"></script>
-    <script src="bower_components/angular-form-messages/src/bootstrap/afFeedback.js"></script>
-    <script src="bower_components/angular-form-messages/src/bootstrap/afFieldState.js"></script>
-    <script src="bower_components/angular-form-messages/src/bootstrap/afMessages.js"></script>
-    <script src="bower_components/angular-form-messages/src/bootstrap/templateCache.js"></script>
-    <script src="bower_components/angular-form-messages/src/afField.js"></script>
-    <script src="bower_components/angular-form-messages/src/afSubmit.js"></script>
-    <script src="bower_components/angular-form-messages/src/afMessageLabel.js"></script>
-    <script src="bower_components/angular-form-messages/src/afSubmitButton.js"></script>
-    <script src="bower_components/angular-form-messages/src/afMessageService.js"></script>
-
+    <script src="bower_components/angular-form-messages/dist/angular-form-messages.min.js"></script>
```

Comment:
Yes I agree on having easy setup after checking out this project. I will find a solution for the developer situation.